### PR TITLE
Add webhook bot

### PR DESCRIPTION
**Pull request by dplewis**

### CODE REVIEW

File: release_benchmarks.sh [hunk 1]
```diff
@@ -34,3 +31,11 @@ data+=("$RESULT")
 
 # Save to data.json
 printf "%s\n" "${data[@]}" > $OUTPUT_FILE
+
+cd docs
+
+git config user.name "Travis CI"
+git config user.email "github@fb.com"
```
Comment (TomWFox) [inline release_benchmarks.sh]: Why does this use an old fb bot?

Comment (dplewis) [reply release_benchmarks.sh]: I don’t know the traviscibot email, I’ll try to change it.

Comment (dplewis) [reply release_benchmarks.sh]: Or I could just leave my face on there

Comment (TomWFox) [reply release_benchmarks.sh]: I would think its fine if it's your face, or maybe we could create a new parse community bot account

Comment (dplewis) [reply release_benchmarks.sh]: Do you know what website theme is used for docs and blog?

Comment (TomWFox) [reply release_benchmarks.sh]: not sure

Comment (TomWFox) [reply release_benchmarks.sh]: For the website it says -

**Pull request merged**